lib/buffer.js

```diff
@@ -624,10 +624,30 @@ function stringSlice(buf, encoding, start, end) {
   throw new ERR_UNKNOWN_ENCODING(encoding);
 }
 
-Buffer.prototype.copy =
-  function copy(target, targetStart, sourceStart, sourceEnd) {
-    return _copy(this, target, targetStart, sourceStart, sourceEnd);
-  };
+function validateOffsetType(val, name) {
+  if (val) {
+    if (typeof val !== 'number') {
+      throw new ERR_INVALID_ARG_TYPE(name, ['Number'], val);
```

Comment:
```suggestion
      throw new ERR_INVALID_ARG_TYPE(name, 'number', val);
```

This resolves https://github.com/nodejs/node/pull/23840/files#r227630142